### ISSUE TITLE
Small suspected bug in error message 

### ISSUE THREAD

**Issue by CarlOhlsson** [microsoft/navcontainerhelper#1744]
**Describe the issue**
Think I stepped on a small bug while testing "New-BcAuthContext" today. If I enter a ClientID that is incorrect just like my example the error message is not what I am expecting. The error message state that a property ‘UserName’ cannot be found. Inspecting the code in New-BcAuthContext.ps1 on line 102 it can be seen a reference is made to $credential.UserName. Looks like a copy paste error from the PSCredential section of the code, should instead show the real error message.

**Scripts used to create container and cause the issue**
```
Connect-AzAccount
$vaultName = "BuildVariables"
$ClientID = "1337"
$ClientSecret = Get-AzKeyVaultSecret -VaultName $vaultName -Name "AADDelegatedAdminClientSecret"
$authcontext4 = New-BcAuthContext `
    -clientID $ClientID `
    -clientSecret $ClientSecret.SecretValue
```

**Full output of scripts**
```
Connect-AzAccount
$vaultName = "BuildVariables"
$ClientID = "1337"
$ClientSecret = Get-AzKeyVaultSecret -VaultName $vaultName -Name "AADDelegatedAdminClientSecret"
$authcontext4 = New-BcAuthContext `
    -clientID $ClientID `
    -clientSecret $ClientSecret.SecretValue

BcContainerHelper version 2.0.5
Attempting authentication to https://api.businesscentral.dynamics.com/.default using clientCredentials...
The property 'UserName' cannot be found on this object. Verify that the property exists.
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\2.0.5\BcSaaS\New-BcAuthContext.ps1:102 char:13
+             Write-Host -ForegroundColor Red $exception.Message.Replac ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : PropertyNotFoundStrict
```

**Comment by freddydk** [comment]
You are absolutely right, it hides the real error by introducing a new.
Thanks for reporting

**Comment by freddydk** [comment]
Shipped in BcContainerHelper 2.0.6
